cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesExport.java

```diff
@@ -252,6 +253,12 @@ public String getFormat() {
             "Model description",
             null);
 
+    private static final Parameter EXPORT_TRANSFORMERS_WITH_HIGHEST_VOLTAGE_AT_END1_PARAMETER = new Parameter(
+            EXPORT_TRANSFORMERS_WITH_HIGHEST_VOLTAGE_AT_END1,
+            ParameterType.BOOLEAN,
+            "Export transformers with highest voltage at end1",
+            CgmesExportContext.EXPORT_POWER_FLOWS_FOR_SWITCHES_DEFAULT_VALUE);
+
```

Comment:
we should define a separate variable for the default value of this parameter

Reply:
Yes, right.